testing/marketplace/deployer_envsubst_base/standard/test

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -eox pipefail
+
+test_directory=$(mktemp --dry-run)
+cp -r $(dirname "$0") "$test_directory"
+cd "$test_directory"
+
+image="$REGISTRY/testing/deployer_envsubst/1:$MARKETPLACE_TOOLS_TAG-$TEST_ID"
+
+docker build \
+    --build-arg "MARKETPLACE_TOOLS_TAG=$MARKETPLACE_TOOLS_TAG" \
+    --tag "$image" \
+    .
+
+docker push "$image"
+
+docker run \
+    "gcr.io/cloud-marketplace-tools/k8s/dev:$MARKETPLACE_TOOLS_TAG" \
+    cat /scripts/dev \
+  > dev
+chmod a+x dev
+
+export MARKETPLACE_TOOLS_IMAGE="gcr.io/cloud-marketplace-tools/k8s/dev"
```

Comment:
need export the TAG (instead)

Reply:
I think `MARKETPLACE_TOOLS_TAG` is already exported by caller.

`MARKETPLACE_TOOLS_IMAGE` is not necessary - stale relic of a broken approach.

Thanks for pointing it out!